editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1440,7 +1444,8 @@ Error EditorSceneImporterGLTF::_parse_materials(GLTFState &state) {
 		if (d.has("name")) {
 			material->set_name(d["name"]);
 		}
-		material->set_flag(StandardMaterial3D::FLAG_ALBEDO_FROM_VERTEX_COLOR, true);
+		//don't do this here only if vertex color exists
```

Comment:
since the albedo code was touched i have to reverify that the test cases don't regress https://github.com/godotengine/godot/issues/21087#issuecomment-667750227

Reply:
This flag is less optimal if turned on, so I only make it on when needed